app/helpers/wallet.js

```diff
@@ -0,0 +1,59 @@
+import monnifyConfig from '../config/monnify';
+import getAxiosInstance from './monnifyHttp';
+import apiEndpoint from '../config/apiEndpoint';
+
+/**
+ * @method  reserveAccount
+ * @param {string} name - The name of the user to reserve an account for
+ * @param {string} email - The user's email
+ * @returns {json} Formatted JSON server response
+ */
+async function reserveAccount(name, email) {
+  const accountDetails = {
+    accountReference: `${name}'s reserved-account`,
+    accountName: name,
+    currencyCode: 'NGN',
+    contractCode: monnifyConfig.contractCode,
+    customerEmail: email,
+    customerName: name,
+  };
+
+  const axiosInstance = await getAxiosInstance();
+
+  const { data } = await axiosInstance.post(
+    apiEndpoint.RESERVE_ACCOUNT_NUMBER,
```

Comment:
Seems we are not saving on database service any longer. I believe the data saving here should be done on this service itself.

Reply:
Just a quick check on our architecture. I feel we should have a service to handle all operations on the wallets. I'll drop a message on the slack workspace.

Reply:
Ok @chuxmykel . We should make out time to discourse this.

Reply:
It's all right sir. Whenever you're ready.